js/components/Breakpoints.js

```diff
@@ -1,24 +1,40 @@
 const React = require("react");
 const dom = React.DOM;
+const { connect } = require("react-redux");
+const { getSources } = require("../queries");
 
-function Breakpoints({ breakpoints }) {
+require("./Breakpoints.css");
+require("../lib/variables.css");
 
-  function onResumeClick() {
-    console.log('click')
-    gThreadClient.resume();
-  }
+const Breakpoints = React.createClass({
```

Comment:
Breakdpoints was refactored to be a react class because it needed to use connect, does that make sense? Also, should we be transitioning to es6 class syntax e.g. `class Breakpoints extends React.Component { }`